js/dist-typings/admin/components/StatusWidget.d.ts

```diff
@@ -1,5 +1,6 @@
 export default class StatusWidget extends DashboardWidget {
     items(): ItemList<any>;
+    toolsItems(): ItemList<any>;
```

Comment:
This file will be automatically updated. Could we remove the changes to it from this PR, please?

Reply:
Ok, in the new commits it is no longer there